ex/loqui/lib/loqui/client.ex

```diff
@@ -1,5 +1,89 @@
 defmodule Loqui.Client do
+  @moduledoc """
+  # A high performance client for the Loqui protocol
+
+  This client exposes five public functions, `start_link`, `request`, `push`, `ping` and `close`.
```

Comment:
Sometimes, having a ping function is a good sanity check that you got things working at a low level.